contracts/dao/FeiDAO.sol

```diff
@@ -78,6 +91,33 @@ contract FeiDAO is
         emit ProposalThresholdUpdated(oldProposalThreshold, newProposalThreshold);
     }
 
+    /// @notice one-time option to roll back the DAO to old GovernorAlpha
+    /// @dev admin-only, and expires after the deadline. This function is here as a fallback in case something goes wrong.
+    function __rollback(uint256 eta) external {
+        require(msg.sender == _admin, "FeiDAO: caller not admin");
+        // Deleting admin prevents multiple triggers of this function
+        _admin = address(0);
+
+        require(eta <= ROLLBACK_DEADLINE, "FeiDAO: rollback expired");
+        _eta = eta;
+
+        ICompoundTimelock _timelock = ICompoundTimelock(payable(timelock()));
+        _timelock.queueTransaction(timelock(), 0, "setPendingAdmin(address)", abi.encode(OLD_GOVERNOR), eta);
```

Comment:
Is abi.encode here correct or should it be encodePacked? (Are you taking this from an existing example?)

Reply:
This is forked from compound's GovernorAlpha logic. encode is how you format something for an actual external call, encodePacked ignores padding and is used for hashing.